exasol_integration_test_docker_environment/cli/common.py

```diff
@@ -4,14 +4,15 @@
 import traceback
 from datetime import datetime
 from pathlib import Path
-from typing import Callable, Tuple, Set
+from typing import Callable, Tuple, Set, Optional
```

Comment:
Generally I like and use the `black` way of imports

```python
from typing import (
   Callable,
   Tuple,
   Set,
   Optional
)
```
Simplifies merging etc. also if the order gets changed or so.

Reply:
Done